website/.yarnrc.yml

```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```

Comment:
vat is this

Reply:
...more actionably, can we move this (and the yarn.lock diff) to its own PR?

Reply:
This is something Yarn v3 uses for its local node_modules cache, which is a folder of zip files of the project's dependencies that can be optionally checked into source control. I imagine some people use this feature for dependable static builds without needing to fetch dependencies from npm, but I added the `.yarn` cache to the `.gitignore` in a previous commit, so that only shows up locally.

Apparently I'm the only one that uses Yarn v3, so this hasn't come up before. I'll pull these changes out into another PR.